cmd/agent/app/integrations.go

```diff
@@ -291,8 +306,10 @@ func pip(args []string) error {
 			pipCmd.Dir, _ = executable.Folder()
 		}
 	} else {
-		if inToto {
+		if inToto && withoutTuf {
 			return errors.New("--in-toto conflicts with --no-tuf")
+		} else if inToto && local {
```

Comment:
Let's say `withoutTuf=false` and `local=true`. Wouldn't the `else if inToto && local` not get executed?

Reply:
Well it shouldn't be executed then. Unless we specify that we want to use in-toto.
This logic should go away anyway when we enable in-toto by default.